checkov/terraform/plan_runner.py

```diff
@@ -181,11 +181,12 @@ def _get_graph_report(
             return graph_report
         return self.get_graph_checks_report(root_folder, runner_filter)
 
-    def _create_terraform_graph(self) -> TerraformLocalGraph:
+    def _create_terraform_graph(self, runner_filter) -> TerraformLocalGraph:
         graph_manager = TerraformGraphManager(db_connector=self.db_connector)
         tf_local_graph, _ = graph_manager.build_graph_from_source_directory(
             self.repo_root_for_plan_enrichment,
-            render_variables=True
+            render_variables=True,
+            download_external_modules=runner_filter.download_external_modules
```

Comment:
I see runner_filter can be defined as None in the parent function, so seems we should check that before accessing this field